satellite/repair/repairer/ec.go

```diff
@@ -405,9 +410,13 @@ func (ec *ECRepairer) putPiece(ctx, parent context.Context, limit *pb.AddressedO
 	// to slow connection. No error logging for this case.
 	if ctx.Err() == context.Canceled {
 		if parent.Err() == context.Canceled {
-			ec.log.Info("Upload to node canceled by user", zap.String("NodeID", storageNodeID.String()))
+			ec.log.Info("Upload to node canceled by user",
+				zap.Binary("Segment", []byte(path)),
+				zap.String("NodeID", storageNodeID.String()))
```

Comment:
I think you can use stringer here and below instead of the storageNodeID.String() right?

Reply:
Yea I guess so. I have seen that in some other log lines. Let me correct it while I am on it.